straxen/plugins/event_processing.py

```diff
@@ -534,7 +532,18 @@ class EventPositions(strax.Plugin):
                 '?version=ONLINE&run_id=plugin.run_id',
         help='Electron drift time from the gate in ns',
         cache=True)
-    
+
+    fdc_map = straxen.URLConfig(
+        infer_type=False,
+        help='3D field distortion correction map path',
+        default_by_run=[
+            (0, pax_file('XENON1T_FDC_SR0_data_driven_3d_correction_tf_nn_v0.json.gz')),  # noqa
+            (first_sr1_run, pax_file('XENON1T_FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part1_v1.json.gz')), # noqa
+            (170411_0611, pax_file('XENON1T_FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part2_v1.json.gz')), # noqa
+            (170704_0556, pax_file('XENON1T_FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part3_v1.json.gz')), # noqa
+            (170925_0622, pax_file('XENON1T_FDC_SR1_data_driven_time_dependent_3d_correction_tf_nn_part4_v1.json.gz'))], # noqa
+    )
```

Comment:
maybe we can even make the default_by_run a legacy url config

Reply:
I guess i can do that. This would change xenont1t lineages though, not sure its worth the trouble

Reply:
aren't you doing so already because of the pulse-processing changes?

anyway, 1T is kind of a freak chain now and we've already broken it some time ago with some of the corrected area plugins.


The added benifit is that we could enforce our own "specify options once, at the plugin" policy better, see https://github.com/AxFoundation/strax/issues/400

Reply:
Right, good point. Ok ill change it

Reply:
Done.